### PR TITLE
[pubsub][rabbitmq]add support for binding multiple routingkeys at the same time

### PR DESCRIPTION
**Pull request by saber-wang**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1741

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#2465

### CODE REVIEW

**Review by Taction — APPROVED** [review]
LGTM

File: pubsub/rabbitmq/rabbitmq.go [hunk 1]
```diff
@@ -328,16 +328,20 @@ func (r *rabbitMQ) prepareSubscription(channel rabbitMQChannelBroker, req pubsub
...
+		metadataRoutingKey = val
 	}
-	r.logger.Infof("%s binding queue '%s' to exchange '%s' with routing key '%s'", logMessagePrefix, q.Name, req.Topic, routingKey)
-	err = channel.QueueBind(q.Name, routingKey, req.Topic, false, nil)
-	if err != nil {
-		r.logger.Errorf("%s prepareSubscription for topic/queue '%s/%s' failed in channel.QueueBind: %v", logMessagePrefix, req.Topic, queueName, err)
+	routingKeys := strings.Split(metadataRoutingKey, ",")
+	for i := 0; i < len(routingKeys); i++ {
+		routingKey := routingKeys[i]
+		r.logger.Infof("%s binding queue '%s' to exchange '%s' with routing key '%s'", logMessagePrefix, q.Name, req.Topic, routingKey)
```
Comment (yaron2) [inline pubsub/rabbitmq/rabbitmq.go:338]: Can this be a `Debugf` log?

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: pubsub/rabbitmq/rabbitmq_test.go [hunk 2]
```diff
@@ -256,6 +256,30 @@ func TestPublishReconnectAfterClose(t *testing.T) {
...
+	assert.Equal(t, 1, broker.connectCount)
+	assert.Equal(t, 0, broker.closeCount)
+
+	topic := "mytopic_routingkeys"
+
+	handler := func(ctx context.Context, msg *pubsub.NewMessage) error {
+		return nil
+	}
+
+	err = pubsubRabbitMQ.Subscribe(pubsub.SubscribeRequest{Topic: topic, Metadata: map[string]string{"routingKey": "keya,keyb,"}}, handler)
```
Comment (berndverst) [inline pubsub/rabbitmq/rabbitmq_test.go:279]:
```suggestion
	err = pubsubRabbitMQ.Subscribe(context.Background(), pubsub.SubscribeRequest{Topic: topic, Metadata: map[string]string{"routingKey": "keya,keyb,"}}, handler)
```

**Review by berndverst — APPROVED** [review]

**Comment by berndverst** [comment]
@saber-wang can you please open a Docs PR to clarify this metadata field can be used to specify multiple routingkeys separated by commas?

**Review by berndverst — APPROVED** [review]

**Comment by saber-wang** [comment]
@berndverst  I updated the docs RP https://github.com/dapr/docs/pull/2465

**Pull request merged**